package.json

```diff
@@ -7,7 +7,8 @@
     "install": "cd kite-web-element && npm i && cd ../kite-api && npm i",
     "test": "cd kite-web-element && npm run build && cd ../kite-api && docker-compose -f ../docker-compose.prod.yml build --no-cache && docker-compose -f ../docker-compose.prod.yml up -d",
     "setupDev": "cd kite-web-element && npm i && npm run build && cd ../kite-api && npm i && docker-compose build --no-cache && docker-compose up -d && cd ../kite-web-element && npm run serve --copy",
-    "setupProd": "cd kite-web-element && npm i && npm run build && cd ../kite-api && npm i && docker-compose -f ../docker-compose.prod.yml build --no-cache && docker-compose -f ../docker-compose.prod.yml up -d"
+    "setupProd": "cd kite-web-element && npm i && npm run build && cd ../kite-api && npm i && docker-compose -f ../docker-compose.prod.yml build --no-cache && docker-compose -f ../docker-compose.prod.yml up -d",
+    "web:serve": "cd kite-web-element && npm i && npm run serve"
```

Comment:
This line, while not necessary, would make life easier for those testing kite-web. It can be removed without detriment to the functionality of the changes made